spec/features/frontend/cart_spec.rb

```diff
@@ -17,21 +17,31 @@
     end
 
     def paypal_script_options
-      script_tag_url = URI(page.find('script[src*="sdk/js?"]', visible: false)[:src])
-      script_tag_url.query.split('&')
+      script_tag_url = page.find('script[src*="sdk/js?"]', visible: false)
```

Comment:
nit: this is no more a URL, having dropped the `[:src]`, also this snippet is repeated enough times that it probably deserves to be extracted, e.g. I could see a `js_sdk_script_tag` helper that is then used directly inside the examples

Reply:
Got it! I split this method into two, and moved it into a spec helper. How's it look now?